### DISCUSSION TITLE
Apple to allow sideloading and 3rd party App stores

### DISCUSSION THREAD

**Discussion by LostInCompilation** [utmapp/UTM#4879] [General]
So I think you all heard the news. Thanks to the EU we'll get sideloading on iOS, yay!
Looks like UTM without any jailbreak.

**Comment by osy** [comment]
Don’t be quick to celebrate. There’s nothing that would force Apple to allow JIT in third party apps (although I heard there may be rules to allow 3rd party browsers which would require JIT). But the real win would be if they allow Hypervisor entitlement.
